### ISSUE TITLE
ml2d fails in TestXmippWorkflow

### ISSUE THREAD

**Issue by yaizar** [I2PC/scipion#1492]
```
00018:   mpirun -np 2 -bynode  `which xmipp_mpi_mlf_align2d`  -i Runs/000384_XmippProtML2D/tmp/input_particles.xmd --oroot Runs/000384_XmippProtML2D/mlf2d_ --nref 1 --limit_resolution 0 20.000000 --sampling_rate 9.896000 --iter 4 --mirror
00019:   XMIPP_ERROR 51:  Error related to numerical calculation.
00020:   Prog_MLFalign2D-ERROR%% Avoid different Q0 values in the CTF parameter files!
00021:   File: software/em/xmipp/libraries/reconstruction/mlf_align2d.cpp line: 452
00022:   XMIPP_ERROR 51:  Error related to numerical calculation.
00023:   Prog_MLFalign2D-ERROR%% Avoid different Q0 values in the CTF parameter files!
00024:   File: software/em/xmipp/libraries/reconstruction/mlf_align2d.cpp line: 452
```

**Comment by yaizar** [comment]
@cossorzano @dmaluenda FYI

**Comment by dmaluenda** [comment]
The ctf_estimation_from_psd.cpp program use an integers assignation to the optimizations parameters. Then, this parameters can be ignored by setting `step(i)=0`. In some moment a new parameters has been add modifying the integer assignation, then the integer assignation to the Q0 was change an we was optimizing Q0 even if Amplitude Contrast was introduced in the import.

Since is a big bug in the ctf estimation, this fix should be also merged to devel branch asap.

Now this is fixed on facilities-fixes [commit 467cb97caf7c4034e4669f1894200c430a542dd5]

**Comment by azazellochg** [comment]
I close this, looks fixed
